src/core/lib/transport/metadata_batch.h

```diff
@@ -330,6 +394,7 @@ struct GrpcRetryPushbackMsMetadata {
   static absl::string_view key() { return "grpc-retry-pushback-ms"; }
   using ValueType = Duration;
   using MementoType = Duration;
+  using CompressionTraits = StableValueCompressor;
```

Comment:
Same question as above. Is this value not expected to change, thus its ok to use stable value compressor ?

Reply:
i think so, but leaving the behavior change for later.